Library/Formula/websockify.rb

```diff
@@ -0,0 +1,70 @@
+require "FileUtils"
+
+class Websockify < Formula
+  homepage "https://github.com/kanaka/websockify"
+  url "https://github.com/kanaka/websockify/archive/v0.6.0.tar.gz"
+  sha1 "f2bffebc9e4d94ad3a329638bee63e899eaad9c0"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    system "make"
+
+    Dir.chdir("other") do
+      system "make"
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # Move this so it is not installed with the other docs
+    move 'docs/websockify.1', 'websockify.1'
+    (share+"man"+"man1").install 'websockify.1'
+    (doc+"websockify").install 'docs'
+
+    # Upstream Bug: Not named '.dylib'
+    lib.install 'rebind.so'
+
+    # Install C version of websockify
+    move 'other/websockify', 'other/websockify-c'
+    bin.install 'other/websockify-c'
+
+    # Clean up before installing client
+    Dir.mkdir 'client'
```

Comment:
The [Python document](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md) points to the [jrnl](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/jrnl.rb) formula as an example, but I appreciate that doesn't cover the ` mkdir `. I'll look at Homebrew's example formula and see if I can broaden it a little for clarity.